tests/testSimpleShape.py

```diff
@@ -63,7 +63,7 @@ def checkMoments(self, dEllipseCore, dCenter, wEllipseCore, wCenter):
         image = lsst.afw.image.MaskedImageF(self.bbox)
```

Comment:
I added documentation, and changed the function name to something more appropriate